src/context/Context.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { createContext, useState, ReactNode } from "react";
+
+export const TechContext = createContext({
+  techClicked: 0,
+  setTechClicked: (index: number) => {},
+});
+
+export const TechProvider: React.FC<{ children: ReactNode }> = ({
+  children,
+}) => {
```

Comment:
```suggestion
export const TechProvider = ({ children }: { children: ReactNode }) => {
```

https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13